themes/govcms/govcms_ui_kit/govcms_ui_kit.info

```diff
@@ -8,7 +8,6 @@ stylesheets[all][] = vendor/owl-carousel/owl.transitions.css
 stylesheets[all][] = dist/css/styles.css
 
 scripts[] = vendor/owl-carousel/owl.carousel.min.js
-scripts[] = vendor/customize-twitter/customize-twitter-1.1.min.js
```

Comment:
@alan-cole , you've removed this js file from being used but it's still in the repo, should it still be there?